releases/release-1.30/README.md

```diff
@@ -0,0 +1,121 @@
+| **Begin [Enhancements Freeze]**                                                        | Enhancements Lead             | [02:00 UTC Friday 9th February 2024 / 18:00 PDT Thursday 8th February 2024](https://everytimezone.com/s/94c1e78b)       | week 5   | [master-blocking], [master-informing]                  |
+| 1.30.0-alpha.2 released                                                                | Branch Manager                | Tuesday 13th February 2024                                                                                                  | Week 6   |                                                        |
+| Docs deadline — Open placeholder PRs                                                   | Docs Lead                     | Thursday 22nd February 2024                                                                                            | week 7   |                                                        |
+| Begin Friday APAC-friendly meetings                                                    | Lead                          | Friday 23rd February 2024                                                                                              | Week 7   |                                                        |
+| **Call for [Exceptions][Exception]**                                                   | Lead                          | Monday 26th February 2024                                                                                              | week 8   |                                                        |
+| Brace Yourself, Code Freeze is Coming                                                  | Comms / Release Signal            | Monday 26th February 2024                                                                                              | week 8   |                                                        |
+| **Begin Feature blog freeze**                                                          | Comms Lead                    | [02:00 UTC Wednesday 27th February 2024 / 18:00 PDT Tuesday 26th February 2024](https://everytimezone.com/s/b23acb87)   | week 8   |                                                        |
+| **Begin [Code Freeze]**                                                                | Branch Manager                | [02:00 UTC Wednesday 6th March 2024 / 18:00 PDT Tuesday 5th March 2024](https://everytimezone.com/s/5056bace)   | week 9   |                                                        |
+| 1.30.0-alpha.3 released                                                                | Branch Manager                | Thursday 7th March 2024                                                                                            | Week 9   |                                                        |
```

Comment:
I have two concerns about this release:

- The time difference between alpha.2 and alpha.3 is almost one month. I'm wondering if we should have one release in between, e.g. on Feb 27
- Given our experience with alpha.3 in 1.29, I think we should consider moving this release for a few days after the code freeze
  - It's safe to expect that we'll have PRs merging on Thursday (the release day) and we'll also get a plenty of exception requests
  - All that increases chances to destabilize the CI signal, making it even more pressuring to both the CI signal team and the release managers

Reply:
I'm happy to have an alpha on Feb 27 so there's less of a gap, but moving alpha.3 (or alpha.4, if we add one on Feb 27) further ahead of code freeze (to March 12, the following Tuesday) will create a conflict with the suggestion of moving beta.0 to the beginning of code freeze.

What do you think about the following:

alpha.2 on Tuesday Feb 13
alpha.3 on Tuesday Feb 27
alpha.4 on Tuesday March 5 (beginning of code freeze)
beta.0 on Tuesday March 12 (week after code freeze)

This gets us an extra alpha, moves beta.0 out of kubecon week, and leaves a one week gap between beta.0 and rc.0 in case something gets screwy and we decide we need a beta.1 during kubecon week. 

I'm also fine moving alpha.3 to Feb 27, not adding an alpha.4, and moving beta.0 to either March 12 or March 5.

Reply:
> I'm also fine moving alpha.3 to Feb 27, not adding an alpha.4, and moving beta.0 to either March 12 or March 5.

+1, I think we don't need alpha.4, given that the CI signal might be shaky at that time and that we might not have the full capacity to deal with it (because of preparations for KubeCon), it seems safer/easier without it.

I'd say we could go with alpha.3 on Feb 27 and beta.0 on March 12, this seems like the safest approach.